.github/workflows/test_runner.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
         username: sebbacon
         password: ${{ secrets.PACKAGE_READ_ACCESS_TOKEN }}
     - name: Generate dummy data
-      run: cohort-extractor generate_cohort --expectations-population=1000
+      run: cohort-extractor generate_cohort --expectations-population=100000
```

Comment:
What was the reason for increasing this?

I made it quite small because we're committing it to git and might make the repo very large.  I appreciated 1000 might be too small for testing - but we should pick a sweet spot

Reply:
This was from a very brief discussion yesterday about where the stata models converge and what size is needed. 100k was picked for convenience I think.....we didn't discuss if we could hit a sweetspot. @alexwalkercebm ?

Reply:
Even 10k has not been large enough for some models to converge (e.g. in the ICS repo I had to change it to 100k). It's dependent on the exact model being run, and also because it's randomly generated _sometimes_ 10k was enough, but it needed 100k to be more reliable.

Reply:
Could we commit a compressed version?